src/02_spec/07_modules/dem_uart/datainterface.rst

```diff
@@ -0,0 +1,50 @@
+Programmer Interface: Data
+--------------------------
+
+The Device Emulation Module only generates one type of event packet: **DEM UART Data packet**
+
+DEM UART Data packet
+^^^^^^^^^^^^^^^^^^^^
+
+A DEM UART Data packet contains one character, that has been sent to a UART interface
+by a CPU, as the only payload. All packets of this type are sent to the DII-Address ``MOD_EVENT_DEST``.
+
+The following fields in the header of the DI packet are set:
+
+- ``FLAGS.TYPE`` is set to ``EVENT``
+- ``FLAGS.TYPE_SUB`` is set to ``0x00``
+
+.. tabularcolumns:: |p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.30\linewidth-2\tabcolsep}|p{\dimexpr 0.50\linewidth-2\tabcolsep}|
+.. flat-table:: Structure of a DEM UART Data packet
+  :widths: 2 3 5
+  :header-rows: 1
+
+The resulting Debug Interconnect packet has the following structure.
+
+  * - Byte
```

Comment:
Packets are given in words, not bytes. You can restrict yourself to describing the payload, the header fields are already described in the general documentation.